implementations/rust/ockam/ockam_api/src/kafka/portal_worker.rs

```diff
@@ -239,17 +258,19 @@ impl KafkaPortalWorker {
     /// used for requests.
     pub(crate) async fn start(
```

Comment:
Having a function like `KafkaPortalWorker::start` is a bit misleading since it starts 2 workers. I'd rather have a simple top level function called `start_kafka_portal` instead.

Reply:
make sense

Reply:
We have an (unwritten) convention of calling a constructor `new` and something that creates and does some additional setup/starts workers is `create`